lib/Mojolicious/Renderer.pm

```diff
@@ -108,6 +108,12 @@ sub render {
   # Inheritance
   my $content = $stash->{'mojo.content'} //= {};
   local $content->{content} = $output =~ /\S/ ? $output : undef if $stash->{extends} || $stash->{layout};
+  if ($stash->{inline_layout}) {
+    @$options{qw(template inline)} = (undef, delete $stash->{inline_layout});
+    $options->{format} = $stash->{format} || $self->default_format;
+    if ($self->_render_template($c, \my $tmp, $options)) { $output = $tmp }
+    $content->{content} //= $output if $output =~ /\S/;
```

Comment:
Don't think this line has test coverage either.

Reply:
Removed too. If I’m not mistaken this is useful only in the event that the processing continues after that. Which can’t happen for now.